example/4-start_nbs_storage.sh

```diff
@@ -21,30 +19,22 @@ fi
 blockstore-client UpdateDiskRegistryConfig --verbose error --input $BIN_DIR/nbs/nbs-disk-registry.txt --proto
 if [ $? -ne 0 ]; then
     echo "Disk registry config was not updated. Did you try to update it for the second time? " \
-      "Consider updating the Version value or adding IgnoreVersion to $BIN_DIR/nbs/nbs-disk-registry.txt"
+        "Consider updating the Version value or adding IgnoreVersion to $BIN_DIR/nbs/nbs-disk-registry.txt"
 fi
 
 start_nbs_agent() {
-  if [ -z "$1" ]; then echo "Agent number is required"; exit 1; fi
+    if [ -z "$1" ]; then
+        echo "Agent number is required"
+        exit 1
+    fi
 
-  diskagentd \
-    --domain             Root \
-    --node-broker        localhost:$GRPC_PORT \
-    --ic-port            $(( $IC_PORT + $1 * 100 )) \
-    --mon-port           $(( $MON_PORT + $1 * 100 )) \
-    --domains-file       $BIN_DIR/nbs/nbs-domains.txt \
-    --ic-file            $BIN_DIR/nbs/nbs-ic.txt \
-    --log-file           $BIN_DIR/nbs/nbs-log.txt \
-    --sys-file           $BIN_DIR/nbs/nbs-sys.txt \
-    --server-file        $BIN_DIR/nbs/nbs-server.txt \
-    --storage-file       $BIN_DIR/nbs/nbs-storage.txt \
-    --naming-file        $BIN_DIR/nbs/nbs-names.txt \
-    --diag-file          $BIN_DIR/nbs/nbs-diag.txt \
-    --auth-file          $BIN_DIR/nbs/nbs-auth.txt \
-    --dr-proxy-file      $BIN_DIR/nbs/nbs-dr-proxy.txt \
-    --load-configs-from-cms \
-    --location-file      $BIN_DIR/nbs/nbs-location-$1.txt \
-    --disk-agent-file    $BIN_DIR/nbs/nbs-disk-agent-$1.txt >logs/remote-da$1.log 2>&1 &
+    source ./prepare_disk-agent.sh || exit 1
```

Comment:
I have final suggestion. We don't really need to source this file 3 time from inside the function. If we move it to the very top, we can in fact remove everything until the first blockstore-client invocation, because it sources `prepare_binaries.sh` itself and also sets both ports and even `BIN_DIR`
```
#!/usr/bin/env bash

source ./prepare_disk-agent.sh || exit 1
blockstore-client UpdateDiskRegistryConfig --verbose error --input $BIN_DIR/nbs/nbs-disk-registry.txt --proto
```

Reply:
I can agree with it from the functionality standpoint. However I think that the way it is now is more verbose and explicit. Moreover, if it happens that ```prepare_binaries.sh``` call or some variables setting are removed from ```prepare_disk_agent.sh``` it will affect this script transitively.

So I think it is worth it to move ```source ./prepare_disk-agent.sh || exit 1``` before ```start_nbs_agent``` function definition, but I don't think that we should move it higher.

Reply:
The point is that we want to minimize duplication in general